### DISCUSSION TITLE
Azienada/e sponsor - fattibilità?

### DISCUSSION THREAD

**Discussion by zerai** [iisgiua/giuaschool#545] [Idee e proposte]
Ciao @trinko,

sono in contatto con una società a cui ho proposto una sponsorizzazione per giuiaschool, loro stanno valutando e sembra che ci possa essere un' interesse (loro operano nel mercato dei testi scolastici medie/superiori).

Eventualmente saresti interessato o ritieni positivo per il progetto una sponsorizzazione in cambio di link/logo nella forma che tu ritieni opportuna?

E' una soluzione già adottata in progetti opensource anche grandi vedi lo stesso symfony [https://symfony.com/backers](https://symfony.com/backers)

Ciao a presto

**Comment by trinko** [comment]
Poiché **giua@school** (non  _giuiaschool_) è un progetto dell'IIS GIUA, per le sponsorizzazioni bisogna seguire l'apposita normativa per le scuole statali, che prevede diverse restrizioni. In particolare va valutato l'eventuale conflitto di interessi, che nel caso specifico mi sembra rilevante, visto che le scuole devono ogni anno adottare i testi scolastici per i propri studenti.
In ogni caso, non è una decisione che spetta a me, ma alla Dirigenza e al Consiglio di Istituto, dopo presentazione formale della proposta.

Ma se lo scopo di tutto ciò è, come immagino, incrementare la produttività del progetto, realizzando magari alcune delle funzionalità richieste da tempo, sappi che non si otterrebbe nessun beneficio. Il motivo è semplice: dietro _giua@school_ non c'è un team di sviluppo, ci sono io e basta. E io più del tanto (troppo) che faccio ora non posso proprio fare. 

Ritengo che sarebbe molto più efficace che una o più scuole finanziassero direttamente lo sviluppo delle funzionalità di cui necessitano, pagando un programmatore di loro fiducia che le realizzi. Il codice sviluppato, nello spirito dell'open source, dovrebbe poi essere presentato come contributo al progetto (tramite la _pull request_), lasciando a me (come _maintainer_ del progetto) il controllo del rispetto degli standard di codifica e di sicurezza.
Questa mi sembra sia l'unica strada percorribile se si desidera realmente che questo registro elettronico possa rispondere alle esigenze di tutte le scuole. Del resto, mi pare ovvio che un progetto open source che non riceve contributi esterni per lo sviluppo, non potrà mai rispondere alle esigenze di molti.
